.gitignore

```diff
@@ -1,2 +1,3 @@
 /composer.lock
 /vendor
+/.idea
```

Comment:
This should just be in your global git ignore rules, otherwise every editor/tool that every developer uses will need adding.
Here's a GitHub article showing how to set one up, if you've not got one already:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Thanks. Good point.